pylearner.ts

```diff
@@ -4776,8 +5755,194 @@ async function testExamples(examples: Example[]) {
   }
 }
 
+async function testAliasingViolationExample(example: Example, errorMessage: string, locStart: number, locEnd: number) {
```

Comment:
Je gebruikt van deze Example-objecten enkel de declarations. In plaats van de aliasing violation test cases in Example-objecten te steken, steek ze in TestCase-objecten met properties `declarations`, `errorMessage`, `locStart`, em `locEnd`. Verwijder de code in deze functie die de statements en de expression nakijkt.